README.md

```diff
@@ -79,6 +79,7 @@ Instructions on using AzureAuth CLI in your applications are available [here](do
 
 # Data Collection Is Off By Default
 No telemetry will be collected unless you set the `AZUREAUTH_APPLICATION_INSIGHTS_INGESTION_TOKEN` environment variable
+or the registry key `ApplicationInsightsKey` under `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\AzureAuth`
```

Comment:
@goagain Is it too late to change the name of the key, or can we request they change the name through another CAB request? I think it would be better if the names of the env var reg key matched. Because the env var is called an Application Insights Ingestion Token, I'd expect/hope the reg key to be named the same, `ApplicationInsightsIngestionToken`. 

And I see that's actually what we called the variable name in the code ;)
* `ApplicationInsightsIngestionTokenRegKeyPath`
* `ApplicationInsightsIngestionTokenRegKeyName`

Reply:
I will ask in CAB workitem

Reply:
Updated